src/setup/cortx_deploy

```diff
@@ -58,6 +58,9 @@ function conf_get {
 DEBUG_FLAG=false
 SOLUTION_CONFIG="/etc/cortx/solution"
 CORTX_CONFSTORE="yaml:///etc/cortx/cluster.conf"
+RELEASE_INFO="yaml:///opt/seagate/cortx/RELEASE.INFO"
+DEPLOY_FLAG=false
```

Comment:
Avoid so many flags. This hampers readability of the code. 

We need to determine an action based on facts. So it should be called 
ACTION=""

If condition A; then ACTION=deploy
if condition B; then ACTION=upgrade
and so on

Reply:
Done